pkg/kubeconfig/kubeconfig.go

```diff
@@ -76,18 +76,10 @@ func BuildKubernetesClientset(s *state.State) error {
 
 	s.RESTConfig.Dial = tunn.TunnelTo
 
-	return errors.WithStack(HackIssue321InitDynamicClient(s))
-}
-
-// HackIssue321InitDynamicClient initialize controller-runtime/client
-// name comes from: https://github.com/kubernetes-sigs/controller-runtime/issues/321
-func HackIssue321InitDynamicClient(s *state.State) error {
```

Comment:
This function is a reminder and explanation of "why it's done this way", please don't remove it

Reply:
It's not required anymore and actually results in a nil pointer exception if k8s was not properly installed.